crates/graphql-wrapping-types/src/lib.rs

```diff
@@ -316,4 +362,20 @@ mod tests {
         wrapping.push_outermost_list_wrapping(list_wrapping);
         assert_eq!(Wrapping::from(wrapping), original);
     }
+
+    #[test]
+    fn test_is_compatible_with() {
```

Comment:
nitpick. should that be test_is_equal_or_more_lenient_than()?

Reply:
touché